services/movie_session.py

```diff
@@ -24,6 +24,11 @@ def get_movie_session_by_id(movie_session_id: int) -> MovieSession:
     return MovieSession.objects.get(id=movie_session_id)
 
 
+def get_taken_seats(movie_session_id: int) -> list:
+    session = MovieSession.objects.get(id=movie_session_id)
+    return list(session.ticket_set.values("row", "seat"))
```

Comment:
You could write this method without retrieving model with `get` by building a request from `Ticket` model for better efficiency